src/main/java/nl/knmi/geoweb/backend/info/MessageConverterInfo.java

```diff
@@ -5,7 +5,7 @@
 
 @Service
 public class MessageConverterInfo {
-	@Value("${info.version}")
+	@Value("${geoweb.messageConverterVersion}")
```

Comment:
This probably still doesn't work. The previously used info.version and  currently used geoweb.messageConverterVersion will not be available internally in this package. In the GeoWeb-BackEnd these properties are extracted from the pom file.
Should we have a version service like this inside this package at all?

Reply:
This is defined in application.yml
This class is never used so it could be removed.